projecteuler/problems/finds.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+from projecteuler.helpers import primenumbers, factors, number_checks
+
+def find_largest_prime_factor(number):
+    primes = primenumbers.get_prime_numbers(0, number)
+    if not primes:
+        return None
+    
+    all_factors = factors.get_factors(number)
+    if not all_factors:
+        return None
+
+    prime_factors = set(primes).intersection(all_factors)
+    return None if not prime_factors else max(prime_factors)
+
+def find_largest_palindromic_sum(min_number_to_sum, max_number_to_sum):
+    for max_number in range(max_number_to_sum, 0, -1):
+        for min_number in range(max_number_to_sum, min_number_to_sum, -1):
+            sum_number = max_number * min_number
+            if number_checks.is_palindromic_number(sum_number):
+                return sum_number
+    return None
+
+def find_smallest_multiple(min_number, max_number):
+    if min_number <= 0 or max_number <= 0:
+        raise TypeError("Min and max inputs must be positive whole numbers")
+    
+    count=1
+    while(True):
```

Comment:
This is slow because the algorithm of checking every possible value is slow. There is a way of cumulatively building the final value by multiplying the accumulated value by the smallest number required to make it divisible by each successive value in the range. *major hints here*

Reply:
I've updated the method, couldn't work out the multiply way, but have got it to around 9 seconds.

Reply:
```python
def smallest_multiple(limit):

    def multiply_by_lowest_int(t, n):
        lowest_multiplier = 1
        for m in range(1, n+1):
            if t * m % n == 0:
                lowest_multiplier = m

        return t * lowest_multiplier

    total = 1
    for n in range(2, limit + 1):
        total = multiply_by_lowest_int(total, n)

    return total
```
something like this ... (untested)